vrclient_x64/vrclient_x64/struct_converters_103a.cpp

```diff
@@ -0,0 +1,100 @@
+#include <stdlib.h>
```

Comment:
If the name of this file is also an SDK version number space should be left for two digits in the "build" number. We happened to only have three versions after we introduced version numbers of this style, but a full beta cycle will definitely result in double digits.

Reply:
Oh, and the SDK version is wrong here too.